pages/models/generations/SM1/GENSAL/index.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: page
+title: GENSAL
+tags: ["#110","Synchronous machine", "RMS", "EMT", "Phasorial", "Rotor", "Stator", "Dynawo", "Opensource", "GENROU", "GENSAL"]
+date: 10/04/2024 
+last-updated: 23/04/2024
+---
+
+# Salient-pole Synchronous Machine
+
+## Context
+
+The model presented is derived from the general model for Synchronous Machine
+
+## Model use, assumptions, validity domain and limitations
+
+The model can be used to perform dynamic studies for salient-pole synchronous machines in the *dq0* reference frame. It is also valid for the round-rotor synchronous machine mode, as it is a particular case of the salient-pole model.
+
+The assumptions made in this model are:
+
+* The three-phase system is balanced
```

Comment:
can you clearly list the difference with GENROU?

Reply:
GENROU and GENSAL models without saturation are equivalent. When there are saturation terms, then the GENSAL only has saturation for one of the axis while GENROU has for both q and d axis